public/index.html

```diff
@@ -1,6 +1,10 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
 
 <head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
this is needed to make the page responsive eventually. Even now it helps make it look nicer on a mobile device